cmd/dep/graphviz.go

```diff
@@ -89,26 +87,24 @@ func (dp gvnode) label() string {
 		label = append(label, dp.version)
 	}
 
-	return strings.Join(label, "\n")
+	return strings.Join(label, "\\n")
 }
 
-// Ensure that the literal string prefix is a path tree match and
-// guard against possibilities like this:
+// isPathPrefix ensures that the literal string prefix is a path tree match and
+// guards against possibilities like this:
 //
 // github.com/sdboyer/foo
 // github.com/sdboyer/foobar/baz
 //
-// Verify that either the input is the same length as the match (in which
-// case we know they're equal), or that the next character is a "/". (Import
-// paths are defined to always use "/", not the OS-specific path separator.)
-func isPathPrefixOrEqual(pre, path string) bool {
-	prflen, pathlen := len(pre), len(path)
-	if pathlen == prflen+1 {
-		// this can never be the case
+// Verify that prefix is path match and either the input is the same length as
+// the match (in which case we know they're equal), or that the next character
+// is a "/". (Import paths are defined to always use "/", not the OS-specific
+// path separator.)
+func isPathPrefix(path, pre string) bool {
+	pathlen, prflen := len(path), len(pre)
+	if pathlen < prflen || path[0:prflen] != pre || pathlen == prflen+1 {
```

Comment:
That last check, `pathlen == prflen+1`, was useful because the use pattern in gps guaranteed that trailing slashes were not a possibility on the `path` input. Does the same apply here? If not, we should drop that check.